### ISSUE TITLE
Unexpected key 'sourceMap' found 

### ISSUE THREAD

**Issue by amazingSurge** [mcasimir/gulp-rollup#63]
Hi,
With new rollup@0.49.3
There is a error.

```
Error in plugin 'gulp-rollup'
Message:
    Unexpected key 'sourceMap' found, expected one of: acorn, amd, banner, cache, context, entry, exports, extend, external, file, footer, format, globals, indent, input, interop, intro, legacy, moduleContext, name, noConflict, onwarn, output, outro, paths, plugins, preferConst, pureExternalModules, sourcemap, sourcemapFile, strict, targets, treeshake, watch
```

Any help?

**Comment by amazingSurge** [comment]
Roullup change numerous options object and CLI arguments in version 0.48
https://github.com/rollup/rollup#0480

see https://gist.github.com/Rich-Harris/d472c50732dab03efeb37472b08a3f32

**Comment by Permutatrix** [comment]
Fixed in v2.15.0. You should use the updated option names with gulp-rollup from now on.
